cookbooks/cdo-secrets/recipes/globals.rb

```diff
@@ -1,4 +1,5 @@
-template "/home/#{node[:current_user]}/#{node.chef_environment}/globals.yml" do
+template 'globals' do
```

Comment:
Did you just have to name this so you can refer to it from cdo_apps.rb#96 above?